sdk/python/feast/infra/utils/snowflake_utils.py

```diff
@@ -53,13 +53,20 @@ def get_snowflake_conn(config, autocommit=True) -> SnowflakeConnection:
     else:
         kwargs = {}
 
+    if "schema" in kwargs:
+        kwargs["schema_"] = kwargs.pop("schema")
+
     kwargs.update((k, v) for k, v in config_dict.items() if v is not None)
     [
         kwargs.update({k: '"' + v + '"'})
         for k, v in kwargs.items()
         if k in ["role", "warehouse", "database", "schema_"]
     ]
-    kwargs["schema"] = kwargs.pop("schema_")
+
+    if "schema" in kwargs:
```

Comment:
should be `"schema_"` here right? Since that's what you're popping the next line?